modules/calib3d/test/test_cameracalibration.cpp

```diff
@@ -2193,6 +2198,10 @@ TEST(CV_RecoverPoseTest, regression_15341)
             points2.at<Point2f>(12) = Point2f(0.0f, 0.0f); // provoke an outlier detection
             Inliers = recoverPose(E, points1, points2, cameraMatrix, R, t, mask);
             EXPECT_EQ(0, (int)mask.at<unsigned char>(12)) << "Detecting outliers in function failed, testcase " << testcase;
+            Inliers2 = recoverPose(points1, points2, cameraMatrix, zeroDistCoeffs, cameraMatrix, zeroDistCoeffs, E3, R, t, RANSAC, 0.999, 1.0, mask);
```

Comment:
> cameraMatrix, zeroDistCoeffs, cameraMatrix, zeroDistCoeffs

Test passes the same camera intrinsics anywhere for this call.
It doesn't validate the added functionality ("different cameras").